src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/controller/UnallocatedCasesController.kt

```diff
@@ -59,8 +60,8 @@ data class UnallocatedCaseCsv(
   @CsvBindByPosition(position = 2)
   var tier: String? = null,
   @CsvBindByPosition(position = 3)
-  @CsvDate("yyyyMMdd HH:mm:ss")
-  var sentence_date: LocalDateTime? = null,
+  @CsvDate("yyyy-MM-dd")
+  var sentence_date: LocalDate? = null,
```

Comment:
I think you can remove this now that the date is coming from source rather than from the event